v-next/hardhat/src/internal/cli/version.ts

```diff
@@ -0,0 +1,5 @@
+import { getHardhatVersion } from "../utils/package.js";
```

Comment:
This file is very short so it doesn't really make sense BUT it is for consistency. The logic for these "special hh commands" should be moved in separate files to reduce the amount of code in the main.ts file